tests/DotenvConfigurationTest.cs

```diff
@@ -1,10 +1,10 @@
 
 /*
- * Written by Warwick Molloy GitHub|@WazzaMo
- * to ensure AspNetCore log configuration can be
- * specified as per AspNetCore documentation
- * and code templates.
- */
+* Written by Warwick Molloy GitHub|@WazzaMo
```

Comment:
I'm happy with the readme entry. I know sometimes it can be hard to track down all the authors of opensource code when a license change is needed some time later.